### PR TITLE
Applied SSP for Dishes Indexpage.

### PR DESCRIPTION
**Pull request by Joeper214**
### For dishes Indes Page Applied the FF:

* Sorting
* Searching
* Pagination

Please Merge PR when you are done reviewing it. Thanks!

### CODE REVIEW

**Comment by Joeper214** [comment]
What's the error for this PR? @pocke

**Comment by pocke** [comment]
https://travis-ci.org/skyarch-networks/skyhopper/jobs/78152336#L2848

Hmm... I don't know the cause...

rspec says the following.

```
     Failure/Error: expect(assigns[:dishes]).to eq dishes

       

       expected: #<ActiveRecord::Relation []>

            got: #<ActiveRecord::Relation []>
```

But I look they are same...

Please try to change assigned `dishes` attribute from empty relation to not empty relation.
Possibly we can  find some diff.


I don't attendance today and tomorrow, but maybe @root-beer attendances. 
If you need to help, please ask to him.
and if you want fast merging, change assignee to him.

**Comment by pocke** [comment]
default

`Project list` on dish page...

File: app/assets/javascripts/dishes.js [hunk 1]
```diff
@@ -11,6 +11,106 @@
...
+             self.data = data.map(function (item) {
+               return {
+                 name: item.name,
+                 detail: item.detail,
+                 status: item.status,
+                 id: item.id,
+               };
+             });
+             self.$emit('data-loaded');
+             var empty = t('projects.msg.empty-list');
```
Comment (pocke) [inline app/assets/javascripts/dishes.js]: `s/projects/dishes/` and update locale

Comment (Joeper214) [inline app/assets/javascripts/dishes.js]: :+1:

**Comment by pocke** [comment]
LGTM. Thanks!

**Pull request merged**